features/admin.claims.v1/pages/local-claims-edit.tsx

```diff
@@ -165,6 +205,7 @@ const LocalClaimsEditPage: FunctionComponent<LocalClaimsEditPageInterface> = (
                         claim={ claim }
                         update={ getClaim }
                         userStores={ userStores }
+                        readOnlyUserStoreNames={ readOnlyUserStoreNamesList }
```

Comment:
Since the readonly user stores list is available via the context, we can access it directly from the component without passing it as a prop. This way, we can reduce prop drilling.
Ref: https://react.dev/learn/passing-data-deeply-with-context